index.bs

```diff
@@ -279,9 +279,19 @@ added to the list of content attributes for video elements.
 The {{autoPictureInPicture}} IDL attribute MUST <a>reflect</a> the content
 attribute of the same name.
 
-The <dfn>autoPictureInPictureElement</dfn> is the video element, among all
-video elements with the {{autoPictureInPicture}} attribute currently set,
-whose {{autoPictureInPicture}} attribute was set most recently. Note that the
+All <a>documents</a> have an associated <dfn>list of Auto Picture-in-Picture
+elements</dfn>, which is an <a>ordered set</a> of {{HTMLVideoElement}} items. It
+is initially empty.
+When the {{autoPictureInPicture}} attribute is added to a video element, the
+video element is first <a>removed</a> then <a>appended</a> into the
+appropriate <a>list of Auto Picture-in-Picture elements</a>.
+When a video element is removed from the {{document}} or its
```

Comment:
Removing from the document removes the element from the list but not appending to the document. Why?

Reply:
I assumed "remove from the document" meant garbage-collected.

Reply:
No, I meant `element.parentElement.removeChild(element);`.

Reply:
As long as the element is still alive, I'm not sure we'd want to remove it from this list even with `element.parentElement.removeChild(element);`.

Reply:
The only way to do that would be to have the list holds a weak reference to the element which I wouldn't know how to express in spec terms. Otherwise, being in that list would prevent the element from being GC'd.